system_noise_covariance: [ 1.e-2, 0, 0, 1.e-2, 0, 1.e-3 ]
measurement_noise_covariance: [ 0.5, 0, 0, 0.5, 0, 0.01 ]
